src/modules/Checkbox/Checkbox.js

```diff
@@ -76,12 +72,10 @@ export default class Checkbox extends Component {
       {fitted: !this.props.label},
       'checkbox'
     );
-
-    const checkboxProps = _.clone(this.props);
-    delete checkboxProps.className;
+    const props = getUnhandledProps(this);
```

Comment:
This is a newer util.  Gets all props not defined in propTypes nor defaultProps.